Source/palette.cpp

```diff
@@ -291,11 +291,13 @@ void PaletteFadeOut(int fr)
  */
 static void CycleColors(int from, int to)
 {
-	SDL_Color col = system_palette[from];
-	for (int i = from; i < to; i++) {
-		system_palette[i] = system_palette[i + 1];
+	{

```

Comment:
the hell is this?

Reply:
a anon scope to avoid overlapping of the two variables called `col`